test/test_sparse_csr.py

```diff
@@ -1537,6 +1537,7 @@ def run_test_block_addmm_addmv(self,
     @parametrize("noncontiguous", [True, False])
     @skipCPUIfNoMklSparse
     @unittest.skipIf(not TEST_SCIPY, "SciPy not found")
+    @skipIfTorchDynamo("raises 'sparse matrix length is ambiguous; use getnnz()'")
```

Comment:
and I'd expect this one to be fixable by breaking properly on sparse tensors

Reply:
Probably. The exception is from scipy though. I'd prefer to kick it down to a follow-up.